src/main/java/com/knu/buga1chuk/interfaces/Animal.java

```diff
@@ -0,0 +1,17 @@
+package com.knu.buga1chuk.interfaces;
+
+public class Animal implements Info {
+    public int id;
```

Comment:
The service classes should not have a state

Reply:
fix